backend-api/src/functions/asyncActiveSession/tokenService/tokenService.ts

```diff
@@ -0,0 +1,117 @@
+import { IKmsAdapter } from "../../adapters/kmsAdapter";
+import { errorResult, Result, successResult } from "../../utils/result";
+
+export class TokenService implements ITokenService {
+  private kmsAdapter: IKmsAdapter;
+
+  constructor(kmsAdapter: IKmsAdapter) {
+    this.kmsAdapter = kmsAdapter;
```

Comment:
Do we feel there's value in passing this down? Is there anything stopping us instantiating the adapter class within the TokenService?

Reply:
We will action this in a follow-up PR